Running test: early_exit
ERROR: premature call to exit()
